src/main/java/pl/edu/zut/mad/schedule/CalendarController.kt

```diff
@@ -0,0 +1,32 @@
+package pl.edu.zut.mad.schedule
+
+import com.ognev.kotlin.agendacalendarview.CalendarController
+import com.ognev.kotlin.agendacalendarview.models.IDayItem
+import org.joda.time.LocalDate
+import java.util.*
+
+
+internal class CalendarController(private val dateListener: DateListener?) : CalendarController {
```

Comment:
Why you use internal access modifier here? The most common usage is in libraries classes that shouldn't be exposed. Is it a case here? 

**This comment applies to all classes.**

Reply:
Indeed, I want to expose only `ScheduleFragment`. Other classes should be invisible.